clients/drcachesim/tests/view_test.cpp

```diff
@@ -329,14 +402,32 @@ run_limit_tests(void *drcontext)
         gen_branch(t1, offs_nop2),
         gen_data(t1, true, 0x42, 4),
     };
+    // To test skipping and limiting we need to use an analyzer which requires
+    // a trace_entry_t version of the above.
+    std::vector<trace_entry_t> records = {
+        test_util::make_thread(t1),
+        test_util::make_pid(/*pid=*/1),
+        test_util::make_marker(TRACE_MARKER_TYPE_VERSION, 3),
+        test_util::make_marker(TRACE_MARKER_TYPE_FILETYPE, 0),
+        test_util::make_marker(TRACE_MARKER_TYPE_CACHE_LINE_SIZE, 64),
+        test_util::make_marker(TRACE_MARKER_TYPE_TIMESTAMP, 1001),
+        test_util::make_marker(TRACE_MARKER_TYPE_CPU_ID, 2),
+        test_util::make_instr(offs_nop1),
+        test_util::make_memref(0x42, TRACE_TYPE_READ, 4),
+        test_util::make_marker(TRACE_MARKER_TYPE_TIMESTAMP, 1002),
+        test_util::make_marker(TRACE_MARKER_TYPE_CPU_ID, 3),
+        test_util::make_instr(offs_jz, TRACE_TYPE_INSTR_UNTAKEN_JUMP),
```

Comment:
The offs_jz is confusing. It should be offs_jcc?

Reply:
This is also in the pre-existing code.  Yes, offs_jcc does seem better, replaced in the original code too.